lldb/include/lldb/Utility/Status.h

```diff
@@ -26,6 +26,36 @@ class raw_ostream;
 
 namespace lldb_private {
 
+class MachKernelError
```

Comment:
These aren't yet used in this PR right? Can we get rid of them?

Reply:
Updated.